pkg/cmd/taskrun/describe_test.go

```diff
@@ -279,3 +279,66 @@ No steps
 
 	test.AssertOutput(t, expected, actual)
 }
+
+func TestTaskRunDescribe_no_taskref(t *testing.T) {
```

Comment:
Is the test with a taskref tested implicitely?

Reply:
Every other test uses a `TaskRef` so that case is covered.